src/main/java/racingcar/model/CarRecord.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.model;
+
+public class CarRecord {
+    private String dashCharacterString;
+
+    public CarRecord(int movingDistance) {
+        dashCharacterString = changeMovingDistanceToDashCharacter(movingDistance).toString();
+    }
+
+    private StringBuilder changeMovingDistanceToDashCharacter(int carMovingDistance) {
+        StringBuilder dashCharacterString = new StringBuilder();
+
+        for(int i = 0; i < carMovingDistance; i++) {
+            dashCharacterString.append("-");
+        }
+
+        return dashCharacterString;
+    }
+
+    public String getDashCharacterString() {
+        return dashCharacterString;
+    }
+}
```

Comment:
View 영역을 위한 객체네요~

```suggestion
public class CarRecord {
    private final String dashCharacterString;

    public CarRecord(int movingDistance) {
        dashCharacterString = "-".repeat(movingDistance);
    }
    
    public String getDashCharacterString() {
        return dashCharacterString;
    }
}
```

위 처럼 간결한 표현이 더 좋을 것 같습니다.